balancer-js/src/modules/relayer/actions.ts

```diff
@@ -0,0 +1,233 @@
+import { BigNumber } from '@ethersproject/bignumber';
+import { MaxInt256 } from '@ethersproject/constants';
+import { BatchSwapStep, SwapType } from '../swaps/types';
+import { Relayer } from './relayer.module';
+import { OutputReference, PoolKind } from './types';
+import { StablePoolEncoder } from '@/pool-stable';
+import { ComposableStablePoolEncoder } from '@/pool-composable-stable';
+
+const poolType2PoolKind = (
```

Comment:
Can you please share a reference so we can read a bit more about PoolType vs PoolKind?
Maybe even add a brief summary so we can get the general idea?
I mean, it just feels like a new way of grouping pools together, but I don't really get why 😬

Reply:
yes, no problem. It's simply a translation from pool's naming to relayer's naming. Pools use poolType and poolTypeVersion and relayer's VaultActions are using enum PoolKind.

defined here:
https://github.com/balancer/balancer-v2-monorepo/blob/9b78879ee3a0dcae57094bdfdae973873e4262cf/pkg/standalone-utils/contracts/relayer/VaultActions.sol#L105